src/app/main/component/events/components/events-list/events-list.component.ts

```diff
@@ -107,6 +115,29 @@ export class EventsListComponent implements OnInit, OnDestroy {
     }
   }
 
+  searchWords(): void {
+    this.searchFilterWords.valueChanges.subscribe((value) => {
+      this.wordsToSearch = value.split(' ');
+      this.sortByWord(this.bufferArray, this.wordsToSearch);
+    });
+  }
+
+  sortByWord(list: EventPageResponceDto[], words: string[]): void {
+    this.eventsList = list.filter((element) => {
+      for (const word of words) {
+        if (element.title.includes(word)) {
+          return true;
+        }
+      }
+      return false;
+    });
+    this.eventsList.length ? (this.noEventsMatch = false) : (this.noEventsMatch = true);
```

Comment:
this.noEventsMatch =  !!this.eventsList.length

Reply:
changed it